soccer/planning/TargetVelPathPlanner.cpp

```diff
@@ -93,26 +104,28 @@ bool TargetVelPathPlanner::shouldReplan(
 // TODO(justbuchanan): Paths aren't dynamically feasible sometimes because it
 // doesn't account for initial velocity
 std::unique_ptr<Path> TargetVelPathPlanner::run(
-    MotionInstant startInstant, MotionCommand cmd,
+    MotionInstant startInstant, const MotionCommand* cmd,
     const MotionConstraints& motionConstraints,
     const Geometry2d::ShapeSet* obstacles, std::unique_ptr<Path> prevPath) {
-    assert(cmd.getCommandType() == MotionCommand::WorldVel);
-
-    // If the start point is in an obstacle, escape from it
```

Comment:
Because it also takes the Pivot Command right now. It doest the comandType check in the lambda